examples/plot_simulate_evoked_newapi.py

```diff
@@ -0,0 +1,150 @@
+"""
+===============
+Simulate dipole using new drives-API
+===============
```

Comment:
I don't know if it's a sphinx-gallery thing but in MNE we always put these '=' to be until the end of the text. Nitpick of course :)

Reply:
actually `make html` vomits a ton of warnings about similar issues elsewhere! Might be worth a separate issue (clean up examples)